tests/foreman/api/test_registration.py

```diff
@@ -70,17 +68,14 @@ def test_host_registration_end_to_end(
     module_target_sat.api.SmartProxy(id=nc.id, organization=[org]).update(['organization'])
     module_target_sat.api.SmartProxy(id=nc.id, location=[module_location]).update(['location'])
 
-    command = module_target_sat.api.RegistrationCommand(
-        smart_proxy=nc,
+    result = rhel_contenthost.api_register(
+        nc,
```

Comment:
Could you cherrypick changes from https://github.com/SatelliteQE/robottelo/pull/15306 ?